frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -66,7 +66,7 @@ export const NameValueEditor = withDragDropContext(class NameValueEditor extends
     });
     return <React.Fragment>
       <div className="row">
-        <div className="col-xs-1 co-empty__header"></div>
+        {(!readOnly && allowSorting) && <div className="col-xs-1 co-empty__header"></div>}
```

Comment:
Unnecessary grouping

```suggestion
        {!readOnly && allowSorting && <div className="col-xs-1 co-empty__header"></div>}
```